plugins/webp-uploads/tests/data/class-image-has-size-source-constraint.php

```diff
@@ -1,11 +1,16 @@
 <?php
 
-namespace PerformanceLab\Tests\Constraint;
+namespace WebP_Uploads\Tests\Constraint;
+
+// Load `Image_Has_Source_Constraint` class if it's not loaded yet.
+if ( ! class_exists( Image_Has_Source_Constraint::class ) ) {
+	require_once __DIR__ . '/class-image-has-source-constraint.php';
+}
```

Comment:
Because autoloading of test helpers was removed?

Reply:
Yes

Reply:
Is this all that is needed then?

```suggestion
require_once __DIR__ . '/class-image-has-source-constraint.php';
```